django/db/models/query_utils.py

```diff
@@ -27,6 +27,23 @@ class InvalidQuery(Exception):
     pass
 
 
+def merge_dicts(dicts):
```

Comment:
This doesn't strike me as an appropriate place given it's not reused anywhere. I'd put it in the file in which it's used, even possibly as a staticmethod next to the method in the class where it's used.